Apps/Examples/Examples/All Examples/ShowHideLayerExample.swift

```diff
@@ -0,0 +1,164 @@
+import MapboxMaps
+
+@objc(ShowHideLayerExample)
+
+public class ShowHideLayerExample: UIViewController, ExampleProtocol {
```

Comment:
App code should never be public since it isn't consumed by any other module. This change unlocks additional compile-time optimizations that are not possible for public constructs.